src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input 
+          type="text"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
```

Comment:
You should add an empty line between multiline children
```suggestion
        </div>
        
        <div class="form-field">
```